sources-docs.js

```diff
@@ -8,6 +8,15 @@ module.exports = [
       patterns: 'en/**'
     }
   },
+  {
+    resolve: `gatsby-source-git`,
```

Comment:
Use single quotes not backticks for all of the changes here

Reply:
Oops yup good call. Copy pasted from the `sources-user.js` file which uses backticks for some reason.